.github/workflows/ruby.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-18.04 # use 18 because version 20 didn't have helvetica font
+    runs-on: ubuntu-latest
```

Comment:
Hmm... tests passed. I guess the latest Ubuntu _does_ have Helvetica?

Reply:
I guess so. I figured I'd try it once before I looked into it. :)